bin/cqlsh.py

```diff
@@ -906,12 +907,30 @@ def cmdloop(self):
                     self.reset_statement()
                     print('')
 
+    def strip_comment_blocks(self, statementtext):
+        comment_block_in_literal_string = re.search('["].*[/][*].*[*][/].*["]', statementtext)
+        if not comment_block_in_literal_string:
+            result = re.sub('[/][*].*[*][/]', "", statementtext)
+            if '*/' in result and not '/*' in result and not self.in_comment:
```

Comment:
this failed the pycodestyle test: https://ci-cassandra.apache.org/job/Cassandra-devbranch/254/testReport/junit/dtest.cqlsh_tests.test_cqlsh/TestCqlsh/test_pycodestyle_compliance/

have fixed it here: https://github.com/thelastpickle/cassandra/commit/b51117fb57948d9b4f67ae9f43cb7e88e4ec1f3f